### DISCUSSION TITLE
mqtt5 support for clients

### DISCUSSION THREAD

**Discussion by sivaNbalusu5** [Yakifo/amqtt#145] [General]
I am just wondering, If we have any plans to implement Mqtt5 for the clients?

**Comment by FlorianLudwig** [comment]
Hi @sivaNbalusu5,

currently, there are no plans for mqtt5. Support would be required to be added in a compatible way that we keep mqtt3.1 compat.

**Comment by ajmirsky** [comment]
@sivaNbalusu5 support for mqtt5 is in progress, updates can be found [Issue 175 - MQTT5](https://github.com/Yakifo/amqtt/issues/174)

There will be an upcoming release of v0.11.5 to create a namespace so that both 3.1.1 and 5.0 can exist side-by-side. And then a 0.12.x release with mqtt5.
